### PR TITLE
Medicine class

### PR DESCRIPTION
**Pull request by 99percentile**

### CODE REVIEW

**Comment by ChesterSng** [comment]
Hi @99percentile, have resolved conflicts :)

File: src/test/java/seedu/address/model/medicine/PricePerUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Test
+    public void constructor_invalidPricePerUnit_throwsIllegalArgumentException() {
+        String invalidPricePerUnit = "";
+        Assert.assertThrows(IllegalArgumentException.class, () -> new PricePerUnit(invalidPricePerUnit));
+    }
+
+    @Test
+    public void isValidPricePerUnit() {
+        // null phone number
```
Comment (blewjy) [inline src/test/java/seedu/address/model/medicine/PricePerUnitTest.java]: Should the comment be null price per unit instead?

Comment (blewjy) [reply src/test/java/seedu/address/model/medicine/PricePerUnitTest.java]: Same for the other two comments below

Comment (99percentile) [reply src/test/java/seedu/address/model/medicine/PricePerUnitTest.java]: changed thanks!

**Pull request closed without merging**